packages/office-addin-debugging/src/commands.ts

```diff
@@ -2,12 +2,27 @@
 // Licensed under the MIT license.
 
 import * as commander from "commander";
+import * as fs from "fs";
 import { logErrorMessage, parseNumber } from "office-addin-cli";
 import * as devSettings from "office-addin-dev-settings";
 import { OfficeApp, parseOfficeApp } from "office-addin-manifest";
-import { AppType, parseAppType, parseDebuggingMethod, startDebugging } from "./start";
+import { AppPlatform, AppType, parseAppPlatform, parseAppType, parseDebuggingMethod, startDebugging } from "./start";
 import { stopDebugging } from "./stop";
 
+function determineManifestPath(platform: AppPlatform, dev: boolean): string {
+    let manifestPath = "";
+    if (process.env.npm_package_config_manifest_location) {
+        const manifestPathToDebug = process.env.npm_package_config_manifest_location;
+        manifestPath = manifestPathToDebug.replace("${flavor}", dev ? "dev" : "prod");
```

Comment:
manifestPathToDebug could be undefined and would cause an exception on .replace(). 

Rather than use manifestPathToDebug as const variable and manifestPath as let variable, you could just use the manifestPath variable.

Better error messages could be provided.

```
let manifestPath = process.env.npm_package_config_manifest_location || "";

manifestPath.replace(...);

if (!manifestPath) {
  throw new Error(`The manifest path was not provided.`);
}

if (!fs.existsSync(manifestPath) {
  throw new Error(`The manifest path does not exist: ${manifestPath}.`);
}

return manifestPath;
```